src/game.h

```diff
@@ -127,6 +127,7 @@ class game
         friend class editmap;
         friend class advanced_inventory;
         friend class main_menu;
+        friend class target_handler;
```

Comment:
There are some private members of `game` used by the target_ui function (last_target). I'm planning to move those to `target_handler` in a following PR which will make the friendship unnecessary.